pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -0,0 +1,99 @@
+import sys
+
+import click
+from doozerlib import exectools
+from pyartcd import exceptions, github
+from pyartcd.runtime import Runtime
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+
+
+class GenPipeline:
+    def __init__(self, runtime: Runtime, nightlies: str, custom: str, in_flight_prev: str, previous: str,
+                 build_version: str, assembly_name: str, build_number: str):
+        self.runtime = runtime
+        self.nightlies = nightlies
+        self.custom = custom
+        self.in_flight_prev = in_flight_prev
+        self.previous = previous
+        self.build_version = build_version
+        self.assembly_name = assembly_name
+        self.build_number = build_number
+
+    async def run(self):
+        # Create doozer cmd arguments
+        nightly_args = ""
+        for nightly in self.nightlies.split(","):
+            nightly_args += f" --nightly {nightly.strip()}"
+
+        cmd = f"--group openshift-{self.build_version} release:gen-assembly --name {self.assembly_name} from-releases {nightly_args}"
```

Comment:
Please use array of string instead of space separated string.

Reply:
Threres a problem with the `exectools.cmd_gather_async` when I tried to use list of strings, so kept as string for now since it works.